### DISCUSSION TITLE
SolidJS support?

### DISCUSSION THREAD

**Discussion by sethmbrown** [mantinedev/mantine#2039] [Feature requests]
Any plans to support SolidJS? Love this library, would love to be able to use it in SolidJS projects well. Thanks!

**Comment by rtivital** [comment]
No, there are no such plans

**Reply by sethmbrown** [reply]
Ok, thank you

**Comment by yf-hk** [comment]
Anyone interested in porting Mantine to SolidJS?

**Reply by pavi2410** [reply]
Currently, it seems there is only [Hope UI](https://github.com/hope-ui/hope-ui) that comes close to Mantine

**Reply by uniuwe** [reply]
@dzcpy I am interested! 

If @rtivital has thumbs up 👍 for us.

**Reply by yf-hk** [reply]
@uniuwe Sorry I'm currently not using SolidJS any more :)
